src/scss/App.scss

```diff
@@ -173,7 +196,199 @@ body {
     border-right: 6px solid transparent;
     margin: 15px;
   }
+} 
+
+///// Search Bar Styles /////
```

Comment:
From `Line 201` and below, the scss shouldn't be included in this PR. 
It looks like you didn't branch off of `master` but rather your other open PR.

All good, Let's remove all the scss from `Line 201` and below, as this code has nothing to do with this task.

Reply:
I thought I had branched off of master but I will double check. As an aside, I had installed that program you recommended, sourcetree, but it was giving issues after setup and I just wanted to get on with my work. I think if I use that git technique you use to check out branching, I should be fine for now.

Reply:
Well, that scss code wasn't from master. I believe what had happened is you saved changes for the scss from another branch (i.e `RAP-27c`) into this branch. Just be mindful of that, no bigs